packages/cardpay-sdk/sdk/reward-pool/base.ts

```diff
@@ -62,7 +63,10 @@ export interface RewardTokenBalance {
   balance: BN;
 }
 
-export type WithSymbol<T extends Proof | RewardTokenBalance> = T & {
+export interface HasTokenAddress {
+  tokenAddress: string;
+}
+export type WithSymbol<T extends HasTokenAddress> = T & {
```

Comment:
Made `WithSymbol` type more generic to accept types that have `tokenAddress` attribute